src/app/core/types/user-entity.interface.ts

```diff
@@ -0,0 +1,13 @@
+export interface IUserEntity {
```

Comment:
don't prefix you interfaces with `I`. More in the article
https://passionfordev.com/you-dont-need-to-prefix-interfaces-in-typescript-with-i/

Also, you can freely remove the class below and just use the interface to create instances. I know that this is in the task but that's unnecessary. You need to use either class or interface in this case, since the class is simple - just use interface.
```ts
export interface User
  id: number;
  firstName: string;
  lastName: string;
}

const users: User[] = [{
  id: 1,
  firstName: 'name`,
  lastName: 'last name'
}];

Reply:
done